### PR TITLE
Adding qryn to the registry and vendor list

### PR DESCRIPTION
**Pull request by lmangani**
Hello Maintainers,

Opening this PR to add qryn to the list of integrations and vendors. Hopefully the template was correctly followed 🤞 

* Link to your distribution, if applicable
  * https://github.com/metrico/otel-collector
* Link that proves that our offering is open source:
  * https://github.com/metrico/qryn/blob/master/LICENSE
* GitHub handle or email address as a point of contact
  * @lmangani 

Thanks in advance!

### CODE REVIEW

File: data/ecosystem/vendors.yaml [hunk 1]
```diff
@@ -209,6 +209,13 @@
   contact: ''
   oss: false
   commercial: true
+- name: qryn
+  distribution: true
+  nativeOTLP: true
+  url: 'https://qryn.cloud'
+  contact: ''
```
Comment (svrnm) [inline data/ecosystem/vendors.yaml:216]:
```suggestion
  contact: 'https://github.com/lmangani'
```

File: data/ecosystem/vendors.yaml [hunk 2]
```diff
@@ -209,6 +209,13 @@
   contact: ''
   oss: false
   commercial: true
+- name: qryn
+  distribution: true
+  nativeOTLP: true
+  url: 'https://qryn.cloud'
```
Comment (svrnm) [inline data/ecosystem/vendors.yaml:215]: Can you please update this with a link to your documentation that details how your offering consumes OpenTelemetry natively via OTLP?

**Review by svrnm — CHANGES_REQUESTED** [review]

File: data/ecosystem/vendors.yaml (hunk 2 above)
Comment (lmangani) [reply data/ecosystem/vendors.yaml:215]: Hello @svrnm thanks for taking the time to review!

We support OTLP ingestion through collector writing to db as well as protocol consumers in our ingestion APIs:

- https://github.com/metrico/otel-collector
- https://qryn.metrico.in/#/support?id=tempo-api /v1/traces (OTLP)
- https://github.com/metrico/qryn/blob/b7c8645fbb308889c0fdebbb0e714abce94fec1c/parsers.js#L175

Let me know if the links provide sufficient information.

Comment (svrnm) [reply data/ecosystem/vendors.yaml:215]: This page is (currently) specifically about vendor's that support OTLP with their backend, so update your URL to the link that points to your ingestion API.

Comment (lmangani) [reply data/ecosystem/vendors.yaml]: Done, thanks!

**Review by svrnm — APPROVED** [review]

**Comment by lmangani** [comment]
Thanks @svrnm 😍

**Pull request merged**